app/client/components/holiday/holidayQuestionsModal.tsx

```diff
@@ -55,10 +55,12 @@ export const HolidayQuestionsModal = (props: HolidayQuestionsModalProps) => (
     </ul>
     <h3>You will need to contact us by phone or email if you...</h3>
     <ul>
-      <li>
-        You want to have your delivery redirected to a temporary address within
-        the same country.
-      </li>
+      {!props.holidayStopFlowProperties.hideDeliveryRedirectionHelpBullet && (
```

Comment:
FYI for those not familiar with this pattern : https://reactjs.org/docs/conditional-rendering.html